src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -425,6 +425,24 @@ public String getQuestionSpecificEditFormHtml(int questionNumber) {
             tableBodyHtml.append(tableRow).append(Const.EOL);
         }
 
+        // Create rubric column options as the last row of the table
+        StringBuilder rubricColumnOptionsFragments = new StringBuilder();
+        String tableOptionsTemplate = FormTemplates.RUBRIC_EDIT_FORM_TABLE_OPTIONS;
+        String tableOptionsFragmentTemplate = FormTemplates.RUBRIC_EDIT_FORM_TABLE_OPTIONS_FRAGMENT;
+
+        for (int i = 0; i < numOfRubricChoices; ++i) {
+            String tableBodyCell = Templates.populateTemplate(tableOptionsFragmentTemplate,
+                    Slots.QUESTION_INDEX, questionNumberString,
+                    Slots.COL, Integer.toString(i),
+                    Slots.ROW, Integer.toString(11)); // Considering the options row to be row number 11. Removable?
```

Comment:
Not very fond of the "magic number" appearing here. If necessary, create a new constant for it.